src/pybind/mgr/dashboard/frontend/src/app/shared/forms/cd-validators.ts

```diff
@@ -683,4 +684,30 @@ export class CdValidators {
       return { invalidAddress: !(addressTest && portTest) };
     };
   }
+
+  /**
+   * Validator function to validate endpoints, allowing FQDN, IPv4, and IPv6 addresses with ports.
+   * Accepts multiple endpoints separated by commas.
+   */
+  static endpointValidator(control: AbstractControl): ValidationErrors | null {
```

Comment:
simply saying `endpoint` is enough since its given from the class name that its a validator

```suggestion
  static endpoint(control: AbstractControl): ValidationErrors | null {
```